javascript/ql/src/Security/CWE-178/CaseSensitiveMiddlewarePath.ql

```diff
@@ -65,7 +65,7 @@ predicate isCaseSensitiveMiddleware(
     arg = call.getArgument(0) and
     regexp.getAReference().flowsTo(arg) and
     exists(string flags |
-      flags = regexp.getFlags() and
+      flags = regexp.tryGetFlags() and
       not RegExp::isIgnoreCase(flags)
```

Comment:
```suggestion
not RegExp::maybeIgnoreCase(flags)
```
I think this should be changed to `maybeIgnoreCase` to match.  

For this particular query I don't think we want to flag the cases where we don't know the flags.